backend/LexBoxApi/GraphQL/UserMutations.cs

```diff
@@ -74,4 +74,18 @@ public async Task<User> DeleteUserByAdmin(DeleteUserByAdminInput input, LexBoxDb
         await user.ExecuteDeleteAsync();
         return User;
     }
+
+    [Error<NotFoundException>]
+    [Error<DbError>]
+    [UseMutationConvention]
+    public async Task<User> DeleteUserByUser(DeleteUserByUserInput input, LexBoxDbContext dbContext, LoggedInContext loggedInContext)
```

Comment:
instead of creating a new mutation let's just use the admin one (renaming it and adjusting how permissions work). Take a look at the asserts used in `ChangeProjectMemberRole` mutation for how we should determine who's allowed to take action. You'll want to add a new assert method to the LexAuthUser for delete permission.